integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/StreamHandoffRDD.scala

```diff
@@ -118,7 +118,7 @@ class StreamHandoffRDD[K, V](
     CommonUtil.setTempStoreLocation(split.index, carbonLoadModel, true, false)
     // use CompactionResultSortProcessor to sort data dan write to columnar files
     val processor = prepareHandoffProcessor(carbonTable)
-    val status = processor.execute(iteratorList)
+    val status = processor.execute(iteratorList, null)
```

Comment:
Please add comment why need to pass null

Reply:
Done.